tsconfig.build.json

```diff
@@ -4,7 +4,7 @@
     "declaration": true,
     "declarationDir": "./dist",
     "jsx": "react",
-    "module": "NodeNext",
+    "module": "CommonJS",
```

Comment:
jest can't handle `ESModules` without some special transforming. I thought this was the path of least resistance to getting  unblocked in expedite-react.